pkg/tui/tui.go

```diff
@@ -143,6 +143,19 @@ func (ui *TUI) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			switch {
 			case key.Matches(msg, ui.common.KeyMap.Help):
 			case key.Matches(msg, ui.common.KeyMap.CmdQuit) && ui.activePage() != sourceConfigurePage:
```

Comment:
What about changing this to check `ui.activePage() == wizardIntroPage` so `q` only quits on the intro page?